MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -58595,6 +58956,700 @@
+      "TimeAvailable": "None",
+      "TimeSetup": "None",
+      "IsTrick": false
+    },
+    {
+      "Id": "Keg Trial Access",
+      "RequiredItems": [],
+      "ConditionalItems": [
+        [
+          "AreaSnowheadTempleClear"
+        ],
+        [
+          "Use Fire Arrow"
+        ],
+        [
+          "Powder Keg Challenge with Hot Spring Water",
+          "Fierce Deity's Mask Anywhere"
+        ]
+      ],
+      "TimeNeeded": "None",
+      "TimeAvailable": "None",
+      "TimeSetup": "None",
+      "IsTrick": false
+    },
+    {
+      "Id": "SHT Pound Switch Puzzle",
+      "RequiredItems": [],
+      "ConditionalItems": [
+        [
+          "MaskGoron"
+        ],
+        [
+          "ItemHookshot"
+        ],
+        [
+          "MaskZora",
+          "Gainer"
+        ],
+        [
+          "Long Jump",
+          "Take Damage"
+        ],
+        [
+          "Fierce Deity's Mask Anywhere"
+        ]
+      ],
+      "TimeNeeded": "None",
+      "TimeAvailable": "None",
+      "TimeSetup": "None",
+      "IsTrick": false
+    },
+    {
+      "Id": "SHT Bridge Room",
+      "RequiredItems": [],
+      "ConditionalItems": [
+        [
+          "MaskGoron",
+          "Magic Meter"
+        ],
+        [
+          "ItemHookshot"
+        ],
+        [
+          "SHT Zora Jumps"
+        ],
+        [
+          "FD Jumps"
+        ],
+        [
+          "Long Jump",
+          "Take Damage",
+          "Any Bomb Bag"
+        ],
+        [
+          "Long Jump",
+          "Take Damage",
+          "MaskBlast"
+        ]
+      ],
+      "TimeNeeded": "None",
+      "TimeAvailable": "None",
+      "TimeSetup": "None",
+      "IsTrick": false
+    },
+    {
+      "Id": "SHT Twin Block Room Climb",
+      "RequiredItems": [],
+      "ConditionalItems": [
+        [
+          "ItemHookshot"
+        ],
+        [
+          "MaskZora"
+        ],
+        [
+          "Fierce Deity's Mask Anywhere"
+        ],
+        [
+          "Goron Bomb Jump",
+          "Take Damage",
+          "Any Bomb Bag"
+        ]
+      ],
+      "TimeNeeded": "None",
+      "TimeAvailable": "None",
+      "TimeSetup": "None",
+      "IsTrick": false
+    },
+    {
+      "Id": "SHT 2nd Locked Door",
+      "RequiredItems": [
+        "SHT Pound Switch Puzzle",
+        "OtherExplosive"
+      ],
+      "ConditionalItems": [
+        [
+          "2 Snowhead Keys"
+        ],
+        [
+          "SHT Bridge Room",
+          "SHT Twin Block Room Climb",
+          "1 Snowhead Key"
+        ]
+      ],
+      "TimeNeeded": "None",
+      "TimeAvailable": "None",
+      "TimeSetup": "None",
+      "IsTrick": false
+    },
+    {
+      "Id": "SHT Compass Room to Twin Block Room",
+      "RequiredItems": [
+        "1 Snowhead Key"
+      ],
+      "ConditionalItems": [
+        [
+          "Use Fire Arrow"
+        ],
+        [
+          "ItemHookshot"
+        ],
+        [
+          "MaskZora",
+          "Gainer"
```

Comment:
I think Zora can jump from the chest. Also human can probably long jump from the chest.

Reply:
The Zora one is redundant because logic was already written to get into the room by doing the bridge room somehow. The human long jump could be added though.

Reply:
Actually, do you want backflipping onto the chest as Zora to be in logic without SHT Zora Jumps? Because if so, it wouldn't be redundant at all to add it to SHT Compass Room to Twin Block Room.

Reply:
Nah i guess not